From ec0364584af411fec5714043fe647f3f7b3b4a6d Mon Sep 17 00:00:00 2001
From: Simon Glass <sjg@chromium.org>
Date: Mon, 14 Aug 2017 13:03:10 -0600
Subject: [PATCH] pylibfdt: Add stdint include to fix uint32_t

Some types used by libfdt.h are not understood the Python module since
they are defined outside it. An example is an attempt to do this:

   import libfdt
   result = libfdt.fdt_node_offset_by_phandle(fdt, phandle)

This results in:

TypeError: in method 'fdt_node_offset_by_phandle', argument 2 of
	type 'uint32_t'

Include the standard integer header file to fix this.

Signed-off-by: Simon Glass <sjg@chromium.org>
---

 pylibfdt/libfdt.i | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/pylibfdt/libfdt.i b/pylibfdt/libfdt.i
index cd1c6a9..c7b79ec 100644
--- a/pylibfdt/libfdt.i
+++ b/pylibfdt/libfdt.i
@@ -52,6 +52,8 @@
 
 %module libfdt
 
+%include <stdint.i>
+
 %{
 #define SWIG_FILE_WITH_INIT
 #include "libfdt.h"
-- 
2.14.0.434.g98096fd7a8-goog

